src/transactions/transactionBuilder/transactionBuilder.ts

```diff
@@ -451,12 +446,7 @@ export function generateSignedTransactionForSimulation(args: InputSimulateTransa
 export function getAuthenticatorForSimulation(publicKey: PublicKey) {
   // TODO add support for AnyMultiKey
   if (publicKey instanceof AnyPublicKey) {
-    if (publicKey.publicKey instanceof Ed25519PublicKey) {
-      return new AccountAuthenticatorSingleKey(publicKey, new AnySignature(new Ed25519Signature(new Uint8Array(64))));
-    }
-    if (publicKey.publicKey instanceof Secp256k1PublicKey) {
-      return new AccountAuthenticatorSingleKey(publicKey, new AnySignature(new Secp256k1Signature(new Uint8Array(64))));
```

Comment:
why remove?

Reply:
It simplifies the logic no?

All AnyPublicKey will use AccountAuthenticatorSingleKey.

No need to check if its Ed25519PublicKey or Secp256k1PublicKey (or KeylessPublicKey for that matter)

Reply:
dont you need different signatures for ed and secp?

Reply:
Oh fair enough I missed that distinction.  In practice though it doesn't matter because this is for simulation.  Like it's just returning an empty signature at the end of the day.

Reply:
I'll just change it back